packages/fast-components-styles-msft/src/flipper/index.ts

```diff
@@ -34,19 +34,34 @@ const styles: ComponentStyles<FlipperClassNameContract, DesignSystem> = (
             width: "40px",
             height: "40px",
             margin: "0",
+            position: "relative",
             color: neutralForegroundRest,
-            background: neutralFillStealthRest,
-            border: `${toPx(
-                designSystem.outlinePatternOutlineWidth
-            )} solid ${neutralOutlineRest(designSystem)}`,
+            background: "transparent",
+            border: `${toPx(designSystem.outlinePatternOutlineWidth)} solid transparent`,
             borderRadius: "50%",
             padding: "0",
-            "&:hover": {
-                background: neutralFillStealthHover,
-                color: neutralForegroundHover,
+            "&::before": {
+                content: "''",
+                top: "-1px",
+                right: "-1px",
+                bottom: "-1px",
+                left: "-1px",
```

Comment:
Can't quite tell without looking at the dom, but does this assume the border is 1px? Should it use the design system value, or any way to get away without needing this offset?

Reply:
this does seem odd - what's this accomplishing?

Reply:
its aligning the Pseudo element with the button container. Without this the button is 40x40 and the Pseudo is 38x38 even with changing the box model. We need to isolate the background so we can apply an 80% transparency and not effect the other elements in `<Flipper />`

Reply:
As Brian mentioned though, I think this has to do with the border which makes assumptions since that maps to a design system value. Can we explore a better solution here? I think using a pseudo element is fine here, but the code itself fails if we change the outlinePatternOutlineWidth.

Reply:
yeah I can map these values to outlinePatternOutlineWidth